doc/manual/source/manual-page.rst

```diff
@@ -458,18 +458,35 @@ These can be requested by providing different commands on the command line.
                   be in one single output file.
 
            json
-                  The list is placed into a JSON object with a single element
-                  *roas* which contains an array of objects with four
+                  The list is placed into a JSON object with up to four
+                  members: *roas* contains the validated route origin
+                  authorizations, *routerKeys* contains the validated 
+                  BGPsec router keys, *aspas* contains the validated
```

Comment:
The `routerKeys` and `aspas` members are in the `json` format too, not just `jsonext`? I'm not seeing it in the output with this branch (yet).